### ISSUE TITLE
Less than perfectly clear README.md

### ISSUE THREAD

**Issue by langdonx** [arijs/prettier-miscellaneous#11]
Would it be reasonable to have README.md cover only information that doesn't exist in prettier's README.md?  It took me a short while (had to comb through commits) to realize that the Options section includes both prettier and prettier-miscellaneous' options.

As well, it's a bit confusing to have README.md refer to the project as prettier, when it appears to be more than just your typical fork.

**Comment by rattrayalex** [comment]
Agreed. A README that only contains the additional options in prettier-misc and a link to the main prettier README may work better.

**Comment by rhengles** [comment]
PR's accepted! ❤️

**Comment by lonix1** [comment]
+1 to above

From what I can tell, these are the extra bits:
- trailingComma
- bracesSpacing
- alignObjectProperties
- spaceBeforeFunctionParen
- cursorOffset
